app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor.js

```diff
@@ -238,7 +237,6 @@ export default function Editor({
           description={description}
           minLength={property.minLength || 0}
           pattern={property.pattern}
-          required={property.mandatory}
```

Comment:
In case we completely want to remove this, in l.188 this is also set for multiline text input (on the quick, I couldn't find a case where this is actually is true currently)

Reply:
Good catch, fixed.